content/infrastructure/installation/install-rippled-on-ubuntu.md

```diff
@@ -91,6 +89,28 @@ Before you install `rippled`, you must meet the [System Requirements](system-req
         sudo setcap 'cap_net_bind_service=+ep' /opt/ripple/bin/rippled
 
 
+9. Optional: configure core dumps
+
+    By default Ubuntu is not configured to produce core files useful for debugging crashes.
+    First run:
+    
+            ulimit -c unlimited
+    
+    Now run `sudo systemctl edit rippled`. The default editor should open and add
+
+        [Service]
+        LimitCORE=infinity
+
+    This creates `/etc/systemd/system/rippled.service.d/override.conf` saving core dumps while not interfering 
+    with the service file provided by the `rippled` package.
+    Core dumps appearing in `/var/lib/apport/coredump/` can be loaded into `gdb` with
```

Comment:
We don't hard-wrap paragraph text in this repo. Also, I think this could be a little easier to follow if reworded:

```suggestion
    This creates the file `/etc/systemd/system/rippled.service.d/override.conf` and configures the OS to save core dumps, without changing the service file provided by the `rippled` package. If your server crashes, you can find the core dump in `/var/lib/apport/coredump/`. To load the core dump for inspection, use a command such as the following:
```